### DISCUSSION TITLE
项目我是部署在新加坡，但更新了最新版本后，提问报错：unsupported_country_region_territory

### DISCUSSION THREAD

**Discussion by xumin323** [ChatGPTNextWeb/NextChat#5100] [Q&A]
我的服务器在新加坡，5月份的版本是正常的，但我更新到7月份的版本后，提问就返回
{
  "error": {
    "code": "unsupported_country_region_territory",
    "message": "Country, region, or territory not supported",
    "param": null,
    "type": "request_forbidden"
  }
}
新加坡应该没被禁呀，请问下是什么原因？

**Comment by sulitu** [comment]
我发现降低版本可以解决 v2.12.4
